src/rdmacm/communication_manager.rs

```diff
@@ -284,6 +285,13 @@ impl EventChannel {
             listener_id,
         })
     }
+
+    /// Returns the file descriptor associated with the [`EventChannel`].
+    pub fn get_fd(&self) -> RawFd {
```

Comment:
Maybe we can provide this method by implementing `AsRawFd` trait? https://doc.rust-lang.org/std/os/fd/trait.AsRawFd.html

Reply:
Probably makes sense, there is only one fd there.